spark/dl/pom.xml

```diff
@@ -142,6 +142,7 @@
                     <systemProperties>
                         <bigdl.disableCheckSysEnv>true</bigdl.disableCheckSysEnv>
                     </systemProperties>
+                    <argLine>-Xmx6g</argLine>
```

Comment:
Sure, I added that. It'll generate a warning in Java 8, but that's not a big deal.